README.rst

```diff
@@ -53,6 +53,35 @@ Now write a ``jenkins.yml`` file and open a PR::
 Many instructions are available. Just ask the bot by commenting ``jenkins:
 help`` in an open PR!
 
+The bot's story
+===============
+
+The bot executes the extensions listed in setup.py's entry point
+'jenkins_epo.bot.extensions'. Each entry point defines a stage number which is
+used by the bot to decide in which order to execute extensions. The
+``jenkins-epo list-extensions`` command outputs the pipeline.
+
+The ``bot.run()`` asyncio coroutine invoked by the ``jenkins-epo bot`` command
```

Comment:
Ici:

*`jenkins-epo bot` creates an isolated context for each branch or PR to test in extension's `self.current`.*